src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <ul className="movies">
+    {movies.map(movie => (
+      <li className="card" data-cy="Movie" key={movie.imdbId}>
```

Comment:
```suggestion
      <li 
        className="card" 
        data-cy="Movie" 
        key={movie.imdbId}
      >
```
Provide a new line for each attributes if their amount >= 3